matroska_tags.xml

```diff
@@ -32,9 +32,9 @@ targeted item taken from another work of art. All tags in this list can be used
     </tag>
     <tag name="COUNTRY" class="Nesting Information" type="UTF-8">
       <description lang="en">The name of the country
-that is meant to have other tags inside (using nested tags) to country specific information about the item.
+that is meant to have other tags inside (using nested tags) to country specific information about the item,
+in the Matroska countries form; see (#country-codes) on country codes.
```

Comment:
`country-codes` is not part of the Tags document (yet)

Reply:
Possible to link from Tags document to the Matroska notes?

Reply:
Well, too difficult for me :). So as an intermediate step I just put full description in Tags document in order to avoid to block the merge of the PR for a file which is currently not intended for final review.
I hope that it is fine enough.